### PR TITLE
RT-9: Disable IPv6 ND Router Advertisement

### PR DESCRIPTION
**Pull request by priyaranjannanda**
This is initial commit forRT-9: Disable IPv6 ND Router Advertisement. 
Resolves https://github.com/openconfig/featureprofiles/issues/1844 .

### CODE REVIEW

**Comment by sachendras** [comment]
Since there is a conflict, please pick RT-9 for this test and include https://github.com/openconfig/featureprofiles/blob/main/testregistry.textproto in your change.

**Comment by priyaranjannanda** [comment]
> Since there is a conflict, please pick RT-9 for this test and include https://github.com/openconfig/featureprofiles/blob/main/testregistry.textproto in your change.

Thanks Sachendras. I have updated the test case number in README and metadata.textproto. Updated the change in  text registry.textproto.
https://github.com/openconfig/featureprofiles/pull/2492

File: feature/interface/ip/ipv6_ND/otg_tests/disable_ipv6_nd_ra_test/disable_ipv6_nd_ra_test.go [hunk 1]
```diff
@@ -0,0 +1,236 @@
+// Copyright 2022 Google LLC
```
Comment (sezhang2) [inline feature/interface/ip/ipv6_ND/otg_tests/disable_ipv6_nd_ra_test/disable_ipv6_nd_ra_test.go:1]: change to 2024

File: feature/interface/ip/ipv6_ND/otg_tests/disable_ipv6_nd_ra_test/disable_ipv6_nd_ra_test.go [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+	if deviations.Ipv6RouterAdvertisementConfigUnsupported(dut) {
+		routerAdvert.SetSuppress(*ygot.Bool(routerAdvertisementDisabled))
+	} else {
+		routerAdvert.SetEnable(*ygot.Bool(false))
+		routerAdvert.SetMode(oc.RouterAdvertisement_Mode_ALL)
+	}
+	return i
+}
+
+// configureOTG configures OTG interfaces to send and recieve ipv6 paxckets
```
Comment (sezhang2) [inline feature/interface/ip/ipv6_ND/otg_tests/disable_ipv6_nd_ra_test/disable_ipv6_nd_ra_test.go:120]: Fix the typo paxckets

**Comment by sezhang2** [comment]
Please also resolve conflicts

**Review by sezhang2 — DISMISSED** [review]
A new deviation ipv6_router_advertisement_config_unsupported is added and tracked by b/316173974.

Please address some minor comments and resolve conflicts.

File: feature/interface/ip/ipv6_ND/otg_tests/disable_ipv6_nd_ra_test/disable_ipv6_nd_ra_test.go (hunk 1 above)
Comment (priyaranjannanda) [reply feature/interface/ip/ipv6_ND/otg_tests/disable_ipv6_nd_ra_test/disable_ipv6_nd_ra_test.go]: Fixed it in the latest commit.

File: feature/interface/ip/ipv6_ND/otg_tests/disable_ipv6_nd_ra_test/disable_ipv6_nd_ra_test.go (hunk 2 above)
Comment (priyaranjannanda) [reply feature/interface/ip/ipv6_ND/otg_tests/disable_ipv6_nd_ra_test/disable_ipv6_nd_ra_test.go]: Fixed it in the latest commit.

**Review by arulkumarsekar — APPROVED** [review]
re approving

**Pull request merged**